src/runtime/GlobalObjectBuiltinTypedArray.cpp

```diff
@@ -843,18 +836,12 @@ static Value builtinTypedArraySet(ExecutionState& state, Value thisValue, size_t
         }
         int srcByteIndex = 0;
         ArrayBufferObject* oldSrcBuffer = srcBuffer;
-        unsigned oldSrcByteoffset = arg0Wrapper->byteOffset();
-        unsigned oldSrcBytelength = arg0Wrapper->byteLength();
-        unsigned oldSrcArraylength = arg0Wrapper->arrayLength();
+        unsigned oldSrcByteOffset = arg0Wrapper->byteOffset();
```

Comment:
IMO we can use size_t instead of unsigned
in 64 bit system, we can get large value over 32bit but llp64 system(linux) uses only use 4 bytes for unsigned

Reply:
@ksh8281 Good catch!
Unfortunately, each member variable (m_byteLength, m_byteOffset, m_arrayLength) is already declared as `unsinged` type value.
IMO this issue needs to be handled in another patch because there are so many cases of it.
I'll take a look at this right after this patch. Thanks

Reply:
Good!